apps/src/templates/instructions/codeReview/CodeReviewDataApi.js

```diff
@@ -7,11 +7,14 @@ export default class CodeReviewDataApi {
     this.scriptId = scriptId;
   }
 
-  getCodeReviewCommentsForProject() {
+  getCodeReviewCommentsForProject(onDone) {
     return $.ajax({
       url: `/code_review_comments/project_comments`,
       method: 'GET',
       data: {channel_id: this.channelId}
+    }).done((data, _, request) => {
+      this.token = request.getResponseHeader('csrf-token');
```

Comment:
Is there an API we have to just get the CSRF token from the backend? That might help clean things up a little here as we could have just an `initialize()` function on the data API and then check against that in all the API calls?

Reply:
Good idea -- I think I saw something like that, let me look...